### PR TITLE
Continue marking machine specific rules

### PR DESCRIPTION
**Pull request by yuumasato**

### CODE REVIEW

File: shared/xccdf/services/nfs.xml [hunk 1]
```diff
@@ -86,17 +86,17 @@ this service should be disabled.
 
 </Group> <!-- end disabling_nfs -->
 
-<Group id="nfs_configuring_all_machines">
+<Group id="nfs_configuring_all_systems">
```
Comment (mpreisler) [inline shared/xccdf/services/nfs.xml]: I am not sure how strongly I feel about this but changing IDs breaks existing tailoring files for our users. We may want to compromise and leave machines in the ID. What do you think?

Comment (shawndwells) [inline shared/xccdf/services/nfs.xml]: Don't have a very strong opinion but a change seems OK given this is a group ID (vs rules, which are more likely to be selected/unselected) and the XCCDF rules within the nfs_configuring_all_machines group don't have ref tags (which would indicate someone along the way cared enough to map to OS SRG or NIST). They don't appear to be actively used in core profiles.

**Comment by rsprudencio** [comment]
I was makring all sysctl checks as machine-only, but I realized that might exist a user-case which makes sense to check kernel parameters inside a container, for example on PaaS environments where you have only access to the container, if we remove this content from containers we might have some false-negatives that impacts users. What do you guys think? I'm leaning toward not marking these rules.

**Comment by jan-cerny** [comment]
@rsprudencio I think that users will most likely scan their containers in offline mode. It means that OpenSCAP will be usually installed on the host. They surely don't want to install OpenSCAP inside all of their hundreds containers, so I think they will not request checking kernel parameters from inside a container.

I'm not an expert on PaaS systems, but the scanner should be integrated in the PaaS system management, run the scan from the host and display scan results in user's dashboard. That's basically what ManageIQ does.

But the user will not be interested in kernel paramaters, because a single kernel is common for all the containers on the host, so that would be a task for the host's administrator, which clearly has access to the host 😀 .

I think that since containers don't contain, I would mark these as machine-only.

File: shared/xccdf/services/nfs.xml (hunk 1 above)
Comment (yuumasato) [inline shared/xccdf/services/nfs.xml]: I noticed that I missed some "Machine" occurrences in this Group.

Breaking existing tailoring files is something I didn't consider when making this change, so I'll revert it.
And I guess users rely much more on title and description of Rule/Group than its ID, so as long as title and description are in good shape users should be safe from misunderstanding it.

**Comment by mpreisler** [comment]
Is this ready to be merged, is this work in progress?

**Comment by rsprudencio** [comment]
@openscap-jenkins test this please

**Comment by rsprudencio** [comment]
@openscap-jenkins retest this please :crossed_fingers:

**Comment by rsprudencio** [comment]
I have no idea why this is failing, I double checked the commit and all I did was add platform after rationales and some typos inside description :disappointed:

**Comment by rsprudencio** [comment]
@openscap-jenkins retest this please

**Comment by rsprudencio** [comment]
@openscap-jenkins test this please

**Comment by redhatrises** [comment]
[retest this please]

**Comment by yuumasato** [comment]
This is ready to be reviewed.

File: shared/xccdf/services/sssd.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@ The System Security Services Daemon (SSSD) is a system daemon that provides acce
 to different identity and authentication providers such as Red Hat's IdM, Microsoft's AD,
 openLDAP, MIT Kerberos, etc. It uses a common framework that can provide caching and offline
 support to systems utilizing SSSD. SSSD using caching to reduce load on authentication
-servers permit offline authentication as well as store extended user user data.
```
Comment (jan-cerny) [inline shared/xccdf/services/sssd.xml]: Nice catch! 👍 💘 🎂

**Comment by jan-cerny** [comment]
LGTM. Great work!

**Pull request merged**